sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/protocol/blob_rest_client.hpp

```diff
@@ -1156,6 +1156,62 @@ namespace Azure { namespace Storage { namespace Blobs {
     int64_t SequenceNumber = 0;
   }; // struct ResizePageBlobResult
 
+  enum class ScheduleBlobExpiryOriginType
+  {
+    Unknown,
+    NeverExpire,
+    RelativeToCreation,
+    RelativeToNow,
+    Absolute,
+  }; // enum class ScheduleBlobExpiryOriginType
+
+  inline std::string ScheduleBlobExpiryOriginTypeToString(
+      const ScheduleBlobExpiryOriginType& schedule_blob_expiry_origin_type)
+  {
+    switch (schedule_blob_expiry_origin_type)
+    {
+      case ScheduleBlobExpiryOriginType::Unknown:
+        return "";
+      case ScheduleBlobExpiryOriginType::NeverExpire:
+        return "NeverExpire";
+      case ScheduleBlobExpiryOriginType::RelativeToCreation:
+        return "RelativeToCreation";
+      case ScheduleBlobExpiryOriginType::RelativeToNow:
+        return "RelativeToNow";
+      case ScheduleBlobExpiryOriginType::Absolute:
+        return "Absolute";
+      default:
+        return std::string();
+    }
+  }
+
+  inline ScheduleBlobExpiryOriginType ScheduleBlobExpiryOriginTypeFromString(
+      const std::string& schedule_blob_expiry_origin_type)
+  {
+    if (schedule_blob_expiry_origin_type == "")
+    {
+      return ScheduleBlobExpiryOriginType::Unknown;
+    }
+    if (schedule_blob_expiry_origin_type == "NeverExpire")
```

Comment:
since you have these strings ("NeverExpire", "RelativeToCreation", etc) in here and also for `ScheduleBlobExpiryOriginTypeToString`, can you add them to Details:: as constants and then just reuse them?

Reply:
That's not easy for auto generator. This auto generator is supposed to be a temporary thing. So why bother wasting more efforts.

Reply:
btw, core team is supposed to support to full-fledged auto generator in the near future. I think this is a very useful feature request.